### PR TITLE
WIP: feat(component): update styling of property pane

### PR DESCRIPTION
**Pull request by markusoelhafen**
This PR replaces #371 . 
I started to update the UI components for alva, to enhance usabilty

## Tasks

- [x] Styling of String Property
- [x] Styling of Enum Property
- [x] Styling of Boolean Property
- [ ] Styling of Asset Property
- [ ] Styling of Unknown Property

### CODE REVIEW

File: src/lsg/patterns/property-items/enum-item/index.tsx [hunk 1]
```diff
@@ -18,22 +19,39 @@ export interface EnumItemProps {
...
+	box-sizing: border-box;
 	width: 100%;
+	padding: ${getSpace(Size.XS)}px;
+	margin-bottom: ${getSpace(Size.M)}px;
+
+	border: 0.5px solid ${colors.grey90.toString()};
+	border-radius: 3px;
+
+	background-color: ${colors.white.toString()};
+	background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' fill='${colors.grey60.toString()}'><path d='M8,4 l8,8 -8,8 z' /></svg>");
```
Comment (markusoelhafen) [inline src/lsg/patterns/property-items/enum-item/index.tsx]: I added the icon as a background image (still wrong rotation) for now, as I haven't found a solution on how to import the icon component for this issue. Is there any way to resolve this @Palmaswell @LKuechler ?

**Comment by tilmx** [comment]
We should combine these commits in the end to
1. update styling of chrome and preview
2. update styling of element pane
3. update styling of property pane

File: src/lsg/patterns/element/index.tsx [hunk 2]
```diff
@@ -113,11 +114,11 @@ const StyledElementLabel = styled.div`
 
 const StyledPlaceholder = styled.div`
```
Comment (tilmx) [inline src/lsg/patterns/element/index.tsx]: @LKuechler Is it possible to display the Placeholder ONLY when an element is being dragged around?

**Comment by tilmx** [comment]
I started an PR for the chrome/element pane updates in #393. I’ll "delete" the commits made here on those topics.

**Comment by marionebl** [comment]
Closing due to #393 superseeding this.

**Comment by tilmx** [comment]
#393 is about the Layout and Element Pane, this one is about the Property Pane, so it should be still valid.

**Comment by tilmx** [comment]
Superseeded by #436

**Pull request closed without merging**